src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String chassis;
```

Comment:
The field 'chassis' should have an access modifier to encapsulate the field properly. Consider using 'private' to restrict direct access.